ros2_control_demo_hardware/src/rrbot_system_multi_interface.cpp

```diff
@@ -238,7 +238,7 @@ return_type RRBotSystemMultiInterfaceHardware::start()
 
   RCLCPP_INFO(
     rclcpp::get_logger("RRBotSystemMultiInterfaceHardware"), "System successfully started! %u",
-    control_lvl_[0]);
+	static_cast<std::underlying_type<integration_lvl_t>::type>(control_lvl_[0]));
```

Comment:
I am fairly disturbed by this line. It's long, it's cryptic, and at the end of the day, it doesn't do anything specific.
Can we `static_cast` to the type the format string is referring to please?
A simple `static_cast<unsigned int>` would do?

Reply:
Done